__tests__/app.test.js

```diff
@@ -43,4 +43,39 @@ describe("GET /api", ()=> {
       expect(body).toEqual(apiEndpoints)
     })
   })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200 accepts an article_id and responds with an array of comments for that given id ', () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body;
```

Comment:
Please add a length check before your forEach ❗ 
I'd also add in something to check the order of the comments - take a look at jest-sorted library for this 👍